src/authenticated/project/ProjectCreatePage.tsx

```diff
@@ -0,0 +1,46 @@
+import { compose } from 'react-apollo';
+import ProjectSection from './ProjectSection';
+import withCreatePage from '../../utils/withCreatePage';
+import {
+  CreateProjectHOC,
+  ProjectSectionFieldsFragment
+} from '../../generated-models';
+
+const ProjectCreatePage = compose(
+  CreateProjectHOC({
+    props: ({ mutate }) => ({
+      onCreate: (newProject: ProjectSectionFieldsFragment) => {
+        if (!newProject.repoName) {
```

Comment:
I feel like at some point this form of validation should be moved into the actual form - not on the create event, but a problem for a later time I suppose.

Reply:
Haha good point